yt_dlp/extractor/altcensored.py

```diff
@@ -0,0 +1,92 @@
+import re
+
+from .common import InfoExtractor
+from .archiveorg import ArchiveOrgIE
+from ..utils import (
+    int_or_none,
+    orderedSet,
+    urljoin,
+    InAdvancePagedList,
+)
+
+
+class AltCensoredIE(InfoExtractor):
+    IE_NAME = 'altcensored'
+    _VALID_URL = r'https?://(?:www\.)?altcensored\.com/(?:watch\?v=|embed/)(?P<id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://www.altcensored.com/watch?v=k0srjLSkga8',
+        'info_dict': {
+            "id": "youtube-k0srjLSkga8",
+            "ext": "webm",
+            "title": "QUELLES SONT LES CONSÉQUENCES DE L'HYPERSEXUALISATION DE LA SOCIÉTÉ ?",
+            'display_id': 'k0srjLSkga8.webm',
+            'release_date': '20180403',
+            'creator': 'Virginie Vota',
+            'release_year': 2018,
+            'upload_date': '20230318',
+            'uploader': 'admin@altcensored.com',
+            'description': 'md5:0b38a8fc04103579d5c1db10a247dc30',
+            'timestamp': 1679161343,
+            'track': 'k0srjLSkga8',
+            'duration': 926.09,
+            'thumbnail': 'https://archive.org/download/youtube-k0srjLSkga8/youtube-k0srjLSkga8.thumbs/k0srjLSkga8_000925.jpg',
+            'yt_views': 30402,
+            'category': 'News & Politics',
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        # Use most data from archive.org (extractor indirection)
+        # But try first to gather a couple of useful information from altcensored
+        webpage = self._download_webpage(url, video_id)
+        yt_views = int_or_none(self._html_search_regex(
+            r'YouTube Views:.*?([0-9,.]+)', webpage, 'view count', default='0').replace(',', ''))
+        category = self._html_search_regex(r'<a href="/category/.*?\n\s+([^<]+)', webpage, 'category')
+        # Hardcoded (very unlikely to need a change in a foreseeable future)
```

Comment:
I don't think the comments are necessary here, and I would slightly tweak the data extraction
```suggestion
        webpage = self._download_webpage(url, video_id)
        yt_views = str_to_int(self._html_search_regex(r'YouTube Views:\s*([\d,]+)', webpage, 'view count', default=''))
        category = self._html_search_regex(r'<a href="/category/\d+">\s*\n?\s*([^<]+)</a>', webpage, 'category', fatal=False)

```
`str_to_int` which automatically replaces the ',' needs to be imported.

Reply:
In all sincerity, your review is extremely valuable and much appreciated, especially with these handy code-suggestions that ease the task (avoiding most of the cost of going back & forth)

But may I kindly suggest to decrease the nit pickiness level (just by 10% or so ?) ;)

In many extractors, there are double-quoted dict keys, imprecise code comments, single-use variable and distinct idioms when it comes to regexp. These are not always _perfect_ or _modern_, just part of individual contributors' habits/preferences (that flake8 itself didn't dare warning of)

(But thank you again for the serious review and all the interesting features from `utils.py` it brought up to my knowledge)

Reply:
> In many extractors, there are double-quoted dict keys, imprecise code comments, single-use variable and distinct idioms when it comes to regexp. These are not always perfect or modern, just part of individual contributors' habits/preferences 

Just fyi these nitpicks would've come from me or from another maintainer if they hadn't come from sepro. The instances of double quotes, unnecessary single-use vars, etc that you mention likely pre-date yt-dlp and would not have made their way into the codebase today

Reply:
@drzraf I know that these code reviews can sometimes feel like pointless nitpicking. Especially when the codebase is littered with the same or worse code style violations. As bashonly pointed out, some of the code in yt-dlp has stayed untouched for 10+ years. There is [work happening](https://github.com/yt-dlp/yt-dlp/pull/7409) to clean this up, but it will take time.

I hope these reviews don't discourage you from contributing in the future. Every contribution is appreciated!

From my side this PR looks good now. At least one core maintainer will also have to review it. Let's see if there is anything we both missed. ;)

(Small tip: you can also apply multiple suggestions at once, from the "Files changed" tab)